armi/apps.py

```diff
@@ -223,6 +231,76 @@ def getParamRenames(self) -> Dict[str, str]:
             self._paramRenames = renames, self._pm.counter
         return renames
 
+    def registerUserPlugins(self, pluginPaths):
+        """
+        Register additional plugins passed in by importable paths.
+        These plugins may be provided e.g. by an application during startup
+        based on user input.
+        Format expected to be a list of full namespaces to plugin classes.
+        There should be a comma between individual plugins and dots representing
+        the importable python namespace. e.g.
+
+        ``armi.stuff.plugindir.pluginMod.pluginCls,armi.whatever.plugMod2.plugCls2``.
+        or on Linux/Unix:
+        ``/path/to/pluginMod.py:pluginCls,/path/to/plugMod2.py:plugCls2```,
+        or on Windows:
+        ``C:\\path\\to\\pluginMod.py:pluginCls,C:\\\\path\\to\\plugMod2.py:plugCls2```,
```

Comment:
are there too many `\\\\`'s in the 2nd path?

Reply:
Python uses `\` as an escape character for strings.

So, in order to build a Python string for a Windows file path, you need to handle the escape character situation.